github-notify.go

```diff
@@ -64,13 +66,80 @@ func main() {
 	for {
 		println("Checking notifications...")
 		notes := retrieveNotifications(client)
-		// if there are mentions push one to pushbullet
-		if len(notes) > 0 {
-			fmt.Printf("Found %d mentions; pushing one.\n", len(notes))
+		notes = reverse(notes);
+		polling := mapToID(notes);
+	
+		// Only keep notifications that still aren't read
+		var unreadCopy = createCopy(unread)
```

Comment:
No need for making this a separate function, I personally find copy(make([]int, len(unred)), unread) clear enough to read and more descriptive.